eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -43,6 +43,7 @@ steps:
   inputs:
     getDropNameByDrop: true
     optimizationInputsDropNamePrefix: OptimizationInputs/$(System.TeamProject)/$(Build.Repository.Name)
+    ShouldSkipOptimize: $(ShouldSkipOptimize)
```

Comment:
Is there any downside to bootstrapping OptProf data? Does it take longer to train or something else?